crates/core/tedge/src/cli/http/command.rs

```diff
@@ -97,6 +97,7 @@ impl HttpCommand {
             while let Some(bytes) = body.next().await {
                 stdout.write_all(&bytes?).await?;
             }
+            stdout.flush().await?;
```

Comment:
If you really want to put this change to test, the entity store test APIs in `ThinEdgeIo.py` like `Register Entity`, `List Entities` etc can be updated to use the `tedge http` command` as done in this commit: https://github.com/thin-edge/thin-edge.io/commit/605ef13034c466000d0ca66789a53bd20bd2979f

Reply:
The change resulted in an additional commit (82b20b5186d7f1f86d3768a71e7a0e8d1f293dfa) due to differences between how curl and tedge http handle status codes > 299 (e.g. error codes) which caused an existing system test to fail